### DISCUSSION TITLE
A rollup plugin for dts-bundle-generator

### DISCUSSION THREAD

**Discussion by rondonjon** [timocov/dts-bundle-generator#208] [General]
Hello @timocov, just a quick heads-up that I like your `dts-bundle-generator` project and find it very helpful and have just wrapped it in a rollup plugin here. I have added a link to your donation page to the README to support your efforts :-)

https://www.npmjs.com/package/rollup-plugin-dts-bundle-generator

**Comment by timocov** [comment]
Many thanks, glad to hear that! ❤️
